torch_em/model/unetr.py

```diff
@@ -215,8 +85,15 @@ def __init__(
     def preprocess(self, x: torch.Tensor) -> torch.Tensor:
         device = "cuda" if torch.cuda.is_available() else "cpu"
 
-        pixel_mean = torch.Tensor([123.675, 116.28, 103.53]).view(-1, 1, 1).to(device)
-        pixel_std = torch.Tensor([58.395, 57.12, 57.375]).view(-1, 1, 1).to(device)
+        if self.use_sam_stats:
+            pixel_mean = torch.Tensor([123.675, 116.28, 103.53]).view(-1, 1, 1).to(device)
+            pixel_std = torch.Tensor([58.395, 57.12, 57.375]).view(-1, 1, 1).to(device)
```

Comment:
I am not sure if this is ever safe to do given that `torch_em` usually pre-normalizes.